tracing/spec/src/main/asciidoc/microprofile-telemetry-tracing-spec.asciidoc

```diff
@@ -426,12 +428,11 @@ In order to enable any of the tracing aspects, the configuration `otel.sdk.disab
 // TODO The hard coded otel-spec-version need to be replaced in future spec releases!
 [IMPORTANT]
 ====
-This is a deviation from the https://github.com/open-telemetry/opentelemetry-specification/blob/v1.14.0/specification/overview.md[OpenTelemetry Specification version 1.14.0] that specifies this configuration property officially, where https://opentelemetry.io[OpenTelemetry] is activated by default!
+This is a deviation from the OpenTelemetry Specification that specifies this configuration property officially, where https://opentelemetry.io[OpenTelemetry] is activated by default!
```

Comment:
The autoconfigure docs also define the default value as `false` here: https://github.com/open-telemetry/opentelemetry-java/tree/v1.28.0/sdk-extensions/autoconfigure#disabling-opentelemetrysdk

Reply:
Dscussed with @Azquelt and we decided to leave it as it is.